docs/examples/use_cases/tensorflow/efficientdet/train.py

```diff
@@ -0,0 +1,239 @@
+            if not tf.io.gfile.exists(ckpt_dir):
+                tf.io.gfile.makedirs(tensorboard_dir)
+            callbacks.append(
+                tf.keras.callbacks.ModelCheckpoint(
+                    filepath=os.path.join(
+                        ckpt_dir, "".join([args.model_name, ".{epoch:02d}.h5"])
+                    ),
+                    save_weights_only=True,
+                )
+            )
+
+        if args.log_dir:
+            log_dir = args.log_dir
+            if not tf.io.gfile.exists(log_dir):
+                tf.io.gfile.makedirs(log_dir)
+            callbacks.append(
+                tf.keras.callbacks.TensorBoard(log_dir=log_dir, update_freq="epoch")
+            )
+
+        model.fit(
+            train_dataset,
+            epochs=args.epochs,
+            steps_per_epoch=args.train_steps,
+            initial_epoch=initial_epoch,
+            callbacks=callbacks,
+            validation_data=eval_dataset if args.eval_during_training else None,
+            validation_steps=args.eval_steps,
+            validation_freq=args.eval_freq,
+        )
+
+        if args.eval_after_training:
+            print("Evaluation after training:")
+            model.evaluate(eval_dataset, steps=args.eval_steps)
+
+        model.save_weights(args.output_filename)
+
+
+if __name__ == "__main__":
+    import argparse
+    from argparse_utils import enum_action
+
+    parser = argparse.ArgumentParser(formatter_class=utils.SmartFormatter)
+    parser.add_argument(
+        "--initial_epoch",
+        type=int,
+        default=0,
+        help="Epoch from which to start training.",
+    )
+    parser.add_argument(
+        "--epochs", type=int, default=300, help="Epoch on which training should finish."
+    )
+    parser.add_argument(
+        "--input_type",
+        action=enum_action(utils.InputType),
+        required=True,
+        help="Input type.",
+    )
+    parser.add_argument(
+        "--images_path",
+        help="Path to COCO images.",
+    )
+    parser.add_argument(
+        "--annotations_path",
+        help="Path to COCO annotations.",
+    )
+    parser.add_argument(
+        "--train_file_pattern",
+        help="TFrecord files glob pattern for files with training data.",
+    )
+    parser.add_argument("--batch_size", type=int, default=64)
+    parser.add_argument(
+        "--train_steps",
+        type=int,
+        default=2000,
+        help="Number of steps (iterations) in each epoch.",
+    )
+    parser.add_argument(
+        "--eval_file_pattern",
+        help="TFrecord files glob pattern for files with evaluation data, "
+        "defaults to `train_file_pattern` if not given.",
+    )
+    parser.add_argument(
+        "--eval_steps",
+        type=int,
+        default=5000,
+        help="Number of examples to evaluate during each evaluation.",
+    )
+    parser.add_argument(
+        "--eval_freq", type=int, default=1, help="During training evaluation frequency."
```

Comment:
This sentence looks strange. Please consider my suggestion.
```suggestion
        "--eval_freq", type=int, default=1, help="Run the evaluation every `eval_freq` iterations."
```

Reply:
Done